### PR TITLE
fixed variable redefinition; tidied up some syntax

### PR DESCRIPTION
**Pull request by jedp**
Addresses issue #1748 - redefinition of a variable in jschannel

### CODE REVIEW

File: resources/static/include_js/include.js [hunk 1]
```diff
@@ -170,9 +170,9 @@
       if (typeof meth === 'string') {
         var delivered = false;
         if (s_boundChans[o] && s_boundChans[o][s]) {
-          for (var i = 0; i < s_boundChans[o][s].length; i++) {
-            if (s_boundChans[o][s][i].win === w) {
-              s_boundChans[o][s][i].handler(o, meth, m);
+          for (var j = 0; j < s_boundChans[o][s].length; j++) {
```
Comment (jedp) [inline resources/static/include_js/include.js:173]: How does j sound?  k is already used elsewhere consistently for dictionary keys.

File: resources/static/include_js/include.js [hunk 2]
```diff
@@ -683,7 +683,7 @@
       var loc = window.location;
       var frames = window.opener.frames;
       var origin = loc.protocol + '//' + loc.host;
-      for (i = frames.length - 1; i >= 0; i++) {
+      for (var i = frames.length - 1; i >= 0; i++) {
```
Comment (jedp) [inline resources/static/include_js/include.js:686]: Looks like this had already been fixed in winchan.js itself

**Comment by shane-tomlinson** [comment]
@jedp - Have these changes been pushed upstream to @lloyd for JSChannel?

**Comment by lloyd** [comment]
pushed upstream: https://github.com/mozilla/jschannel/commit/1ffa1a955d4913899c0a958a0b4b756f04f0590d

r+

**Pull request merged**